tc/autotuner/cuda/autotuner.cc

```diff
@@ -107,9 +107,8 @@ bool skipExecutionOrWarmup<CudaBackend>(
   auto timings = executor.profile(inputs, outputs);
   // 2.a.
   constexpr size_t kCatastrophicPerfFactor = 100;
-  if (bestTimeSoFar < std::numeric_limits<size_t>::max() and
-      timings.kernelRuntime >= std::chrono::microseconds(
-                                   (kCatastrophicPerfFactor * bestTimeSoFar))) {
+  if (bestTimeSoFar < Duration::max() and
```

Comment:
Do we need to expose the fact that a Duration defaults to max()?
We could have an `isSet()` function to perform this check instead.

Reply:
Nah, just made the constructor default to 0 and private